### ISSUE TITLE
Different sounds for connect, disconnect and 'other' notifications

### ISSUE THREAD

**Issue by MailYouLater** [Ylianst/MeshCentral#696]
All of the notifications have the same sound, so when I hear that sound I can tell something happened, but I can't actually tell if an agent just connected or disconnected, if an AMT connected computer just connected or disconnected, or if it's a notification sent by some other means (e.g. sending a message via the `notify` command in an agent's Console tab).

Can we please at least have a different sound for connection than disconnection?  Subtly different sounds for AMT and agent [dis]connection notifications would be nice too, but the main thing I want is to be able to tell if I'm getting a connect or disconnect notification without having to bring up the interface unnecessarily.

Basically I think their should be at least three sounds: 'connect', 'disconnect' and 'other'.
Note: The current notification sound sounds to me like it should be a 'disconnect' sound.

If possible, I think it would be even better to have five sounds: 'AMT connect', 'Agent connect', 'AMT disconnect', 'Agent disconnect' and 'other'.

Thanks for all the work you guys do to make MeshCentral awesome!

**Comment by Ylianst** [comment]
Should not be too difficult... now I need sounds.

**Comment by Ylianst** [comment]
@MailYouLater If you have a few minutes and can select [on this web site](https://notificationsounds.com/) the notification sounds you like for each event, send me the URL of each sound and I will put it in 👍

**Comment by MailYouLater** [comment]
A lot of those sounds could work, but I didn't find any that fit together as a group.  I'll see if I can find something to suggest to you.
Any chance you know where you got the current notification sound?

**Comment by Ylianst** [comment]
Not sure where I got the current sound, it was in a personal archive I think. This said, any license free sound is ok with me.
